tests/ti.blob.addontest.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Blob', function () {
+	var win;
+
+	afterEach(function (done) {
+		if (win) {
+			// If `win` is already closed, we're done.
```

Comment:
Totally unrelated to this PR, but every time I see this in our test suite it makes me shake my head: Why do we not have any property/method to inquire about the current state of a window/view? We should be able to know if a window is closed or not.